app/components/shared/searchable_collection_component/searchable_collection_component.html+checkbox.slim

```diff
@@ -1,4 +1,4 @@
-.collection-component.scrollable class=("collection-component--border" if searchable)
+.collection-component class=("collection-component--border" if searchable) class=("collection-component--scrollable" if scrollable)
```

Comment:
does this work? nice

Reply:
work as in being able to add 2 conditional classes? yes definatley. work as in fixes the problem of scrolling at correct times? i think so but i want to test on dev and havent been able to deploy it there yet

Reply:
yes this works as expected @cpjmcquillan

Reply:
wouldn't it be clearer if we have a method `classes` in the component?